content/docs/user-documentation/configuring.md

```diff
@@ -10,7 +10,69 @@ summary: Configuration options with MicroShift
 modified: "2021-11-03T16:24:11.538+01:00"
 ---
 
-## WIP Content coming soon
+<!-- 
+todo: 
+- logging 
+-->
 
-- config.yaml vs env vars vs command line
-- logging
+## Configuration
+
+Microshift can be configured in three simple ways, in order of precedence: 
+- Commandline arguments
```

Comment:
wdyt, add links to the 3 sections? This is the syntax: [Commandline arguments]({{< ref "#commandline-arguments" >}})

Reply:
Good idea, I'll do that